### ISSUE TITLE
MF:Schematics to configure webpack

### ISSUE THREAD

**Issue by valorkin** [valorkin/module-federation#34]
@skocharian commented on [Wed Sep 30 2020](https://github.com/valorkin/ng-mf-submodules/issues/15)

Once **MF: Resources & scss**  is done we need to automate this with set of  schematics. 

- Create schematics that can generate Webpack configuration based on the angular JSON
-- New project:  @fundamental-ngx/app-shell:ng-add		
-- Existing project: @fundamental-ngx/app-shell:generate schematics 		
- Reorg- existing angular structure to match our ng-mf-submodules

**Comment by valorkin** [comment]
closed as duplicate of #33
